src/Mod/Sketcher/Gui/Resources/icons/obsolete/Sketcher_CreatePointPolyChamfer.svg

```diff
@@ -0,0 +1,515 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why does this PR add so many "obsolete" icons? If the icons are not necessary, they should be removed from the PR.